projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1020 @@
+
+> By default, the search API will only return results that are currently available. In order to also retrieve results that are not available (yet), you'll need to disable the default filters for `availability`. You can reset this default as described in the Default Filters (TO DO: link to guide).
+
+**Applicable on endpoints** 
+
+`/events` `/places` `/offers` 
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+> Note that the `+` sign should be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Retrieve all events that took place in the year 2020:
+```
+GET /events/?q=dateRange:[2020-01-01T00\:00\:00%2B01\:00 TO 2021-12-31T23\:59\:59%2B01\:00]&availableTo=*&availableFrom=*
+```
+
+### \_exists_
+
+With the `_exists_` parameter you can search for the documents that have a specific property in their JSON body.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any property name from the UiTdatabank JSON scheme.
+
+**Example**
+
+Retrieve all events with a description in dutch:
+
+```
+GET /events/?q=_exists_:description.nl
+```
+
+### id
+
+Retrieve events that match a specific uuid with the `id` parameter. It is possible to search by an event id, place id or organizer id.
+
+A specific id can be found by
+-   looking for the value for the `id` property
+```js
+{
+   "id": "75573a64-ddc8-4fd0-8b07-d258939dd74f"
+}
+```
+-   extracting the UUID from the @id property in the JSON-LD projection
+
+```js
+{
+"@id": "https://io.uitdatabank.be/event/75573a64-ddc8-4fd0-8b07-d258939dd74f"
+}
+```
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any uuid of an event, place or organizer.
+
+**Example**
+
+Retrieve the event with id `75573a64-ddc8-4fd0-8b07-d258939dd74f`:
+```
+GET /offers/?q=id:75573a64-ddc8-4fd0-8b07-d258939dd74f
+```
+
+### imagesCount
+
+With the `imagesCount` parameter you can filter organizers by their number of images.
+
+> This parameter can only be used to filter on organizers. To filter events and places by their number of images (mediaObjects), the parameter `mediaObjectsCount` should be used.
+
+**Applicable on endpoints**
+
+`/organizers`
+
+**Possible values**
+-   an integer, e.g. `2`
+-   a range, consisting of two integers (lower and upper bound), e.g. `[1 TO *]`
+
+**Examples**
+
+Retrieve all organizers that have at least one image:
+```
+GET /organizers/?q=imagesCount:>=1
+```
+
+Retrieve all organizers that have 5 to 10 images:
+```
+GET /organizers/?q=imagesCount:[5 TO 10]
+```
+
+### labels
+
+Search for documents that have a specific label with the `labels` parameter.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+A label name (string).
+
+**Example**
+
+Retrieve all events with an UiTPAS label:
```

Comment:
`Paspartoe` is also an UiTPAS label and it will not match this query: https://io.uitdatabank.be/uitpas/labels. It's better to use the `uitpas` URL parameter to find UiTPAS events. 

So maybe use another example here that doesn't imply that this is the way to find UiTPAS events.

Reply:
Or, you could expand the example to also look for the Paspartoe label of course